02_calculator/calculator.rb

```diff
@@ -0,0 +1,35 @@
+def add(a,b)
+    a+b
+end
+def subtract(a,b)
+    a-b
+end
+
+def sum(nums)
+    sum=0
+    for num in nums
+        sum += num
+    end
+    return sum
+end
+
+def mul(nums)
+    mul=1
+    for num in nums
```

Comment:
Find a way to do this without using a for loop. Ruby is a very expressive language, it was designed to be read close to natural english. Try to figure out which methods of the [Enumerable module](https://ruby-doc.org/core-3.0.1/Enumerable.html) can help you.